app/models/activity.rb

```diff
@@ -0,0 +1,4 @@
+# Activity model for customisation & custom methods
+class Activity < PublicActivity::Activity
+  has_many :notifications, inverse_of: :activity, dependent: :destroy
```

Comment:
.... I said do not think...  Means activity is not a notification. It is just a record... @lowjoel  But I can remove that relation..